### PR TITLE
Support for svg conversion

### PR DESCRIPTION
**Pull request by nicolasbeauvais**
This is a first version to add support to svg images conversions.

It handle transparency when converted to png, and by default use a white background when converted to jpg.

The conversion could be moved to it's own package "svg-to-image" to had more feature like handling density etc..

### CODE REVIEW

File: src/FileManipulator.php [hunk 1]
```diff
@@ -25,7 +26,7 @@ public function createDerivedFiles(Media $media)
             return;
         }
 
-        if ($media->type === Media::TYPE_PDF && !class_exists('Imagick')) {
+        if (in_array($media->type, [Media::TYPE_PDF, Media::TYPE_PDF]) && !class_exists('Imagick')) {
```
Comment (freekmurze) [inline src/FileManipulator.php]: one of the two TYPE_PDF should be TYPE_SVG 😄

File: src/FileManipulator.php [hunk 2]
```diff
@@ -109,7 +114,7 @@ public function createTempDirectory() : string
         return $tempDirectory;
     }
 
-    protected function convertToImage(string $pdfFile) : string
+    protected function convertPDFToImage(string $pdfFile) : string
```
Comment (freekmurze) [inline src/FileManipulator.php]: Nitpick: I don't like having extensions as capitals in function names. Could you make the function name camel case?

File: src/FileManipulator.php [hunk 3]
```diff
@@ -118,6 +123,20 @@ protected function convertToImage(string $pdfFile) : string
         return $imageFile;
     }
 
+    protected function convertSVGToImage(string $svgFile) : string
```
Comment (freekmurze) [inline src/FileManipulator.php]: Same here.

**Comment by freekmurze** [comment]
This looks like a great PR, thank you!

I've added some notes in the code. Could you address them?

**Comment by nicolasbeauvais** [comment]
Done :)

**Comment by freekmurze** [comment]
👍

**Comment by freekmurze** [comment]
Changes have tagged as `4.4.0`

**Pull request merged**